packages/mrkaplan/PKGBUILD

```
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=mrkaplan
+pkgver=V1.1.1.r1.g6119344
+_pkgver=${pkgver//.r1.g6119344}
+pkgrel=1
+pkgdesc='Help red teamers to stay hidden by clearing evidence of execution.'
+arch=('any')
+groups=('blackarch' 'blackarch-windows' 'blackarch-exploitation')
+url='https://github.com/Idov31/MrKaplan'
+license=('BSD2')
+depends=()
+makedepends=()
+source=("$pkgname::git+https://github.com/Idov31/MrKaplan.git"
```

```suggestion
source=("$pkgname::git+https://github.com/Idov31/$pkgname.git"
```

it's case-insensitive, don't worry